### PR TITLE
Only completed debugging

### PR DESCRIPTION
**Pull request by letkyle**
# Submitting Week 3 Assessment

### ✅ Completed Files

Mark the following as confirmation that the files in this repository are complete.

- [x] Coding Challenges
- [x] Interview Questions
- [x] React Debugging

### 📝 Feedback

If you would like specific feedback on a particular problem please leave details here:
I still need to finish my coding challenges and add better pseudo coding, and still need to finish interview questions.

### CODE REVIEW

**Review by elysemontano — CHANGES_REQUESTED** [review]
Request changes:  Looks like you are missing researched answers in the interview questions section. Please update this section and I will review this PR.

**Comment by letkyle** [comment]
just finished the researched answers in the interview questions section.

File: code-challenges.test.js [hunk 1]
```diff
@@ -49,19 +68,59 @@ const studyMinutesWeek2 = {
...
+            expect(leastToGreatest(studyMinutesWeek1)).toEqual([15, 15, 20, 30, 30, 60, 90])
+            expect(leastToGreatest(studyMinutesWeek2)).toEqual([10, 15, 20, 45, 60, 65, 100])
+        })
+    })
+
+
 // b) Create the function that makes the test pass.
 
+    const leastToGreatest = (object) => {  
+        let filteredArr = object.filter(value => typeof value === "number")
```
Comment (elysemontano) [inline code-challenges.test.js:83]: Filter only works on arrays and not on objects.  I would recommend checking the resource provided that would help with that step to get your test pass.

File: code-challenges.test.js [hunk 2]
```diff
@@ -49,19 +68,59 @@ const studyMinutesWeek2 = {
...
+    const accSum = (numArr) => {
+        let sumArr = []
+        if (numArr.length > 0) {
+            for (let i = 1; i < numArr.length; i++) {
+                sumArr[0] = numArr[0]
+                sumArr[i] = sumArr[i-1] + numArr[i]
+            }
+        }
+        return sumArr
+    }
```
Comment (elysemontano) [inline code-challenges.test.js:123]: I don't see any pseudocode for this problem and problem 2.

**Review by elysemontano — COMMENTED** [review]
I don't see any pseudocode for problems 2 and 3 in code challenges.  Please update this and I will consider this approved.

**Pull request merged**